static/js/moment-locale-pl.js

```diff
@@ -0,0 +1,125 @@
+//! moment.js locale configuration
+
+;(function (global, factory) {
+   typeof exports === 'object' && typeof module !== 'undefined'
+       && typeof require === 'function' ? factory(require('../moment')) :
+   typeof define === 'function' && define.amd ? define(['../moment'], factory) :
+   factory(global.moment)
+}(this, (function (moment) { 'use strict';
+
+
+    var monthsNominative = 'styczeń_luty_marzec_kwiecień_maj_czerwiec_lipiec_sierpień_wrzesień_październik_listopad_grudzień'.split('_'),
```

Comment:
Why isn't it a list of strings?

Reply:
Seriously, what the fuck? Should we PR this library or whatever it is?

Reply:
https://github.com/moment/moment/blob/develop/locale/pl.js